docs/user-guide/distributeddataclassification.rst

```diff
@@ -7,36 +9,50 @@ Background
 -----------------------------------------
 
 When preparing text data for training a large language model (LLM), it is useful to classify text documents in various ways.
-This enhances the LLM’s performance by enabling it to produce more contextually appropriate and accurate language across different subjects.
+This enhances the LLM's performance by enabling it to produce more contextually appropriate and accurate language across different subjects.
 
 NeMo Curator provides a module to help users run inference with pre-trained models on large volumes of text documents.
 This is achieved by chunking the datasets across multiple computing nodes, each equipped with multiple GPUs, to accelerate the classification task in a distributed manner.
 Since the classification of a single text document is independent of other documents within the dataset, we can distribute the workload across multiple nodes and GPUs to perform parallel processing.
 
-Domain classification and quality classification are two tasks we include as examples within our module.
-Here, we summarize why each is useful for training an LLM.
+Domain classification, quality classification, educational content classification, and content safety models are tasks we include as examples within our module.
+
+Here, we summarize why each is useful for training an LLM:
+
+- **Domain classification** is useful because it helps the LLM understand the context and specific domain of the input text. Because different domains have different linguistic characteristics and terminologies, an LLM's ability to generate contextually relevant responses can be improved by tailoring training data to a specific domain. Overall, this helps provide more accurate and specialized information.
```

Comment:
```suggestion
- The **Domain Classifier** is useful because it helps the LLM understand the context and specific domain of the input text. Because different domains have different linguistic characteristics and terminologies, an LLM's ability to generate contextually relevant responses can be improved by tailoring training data to a specific domain. Overall, this helps provide more accurate and specialized information.
```